app/src/main/res/values/styles.xml

```diff
@@ -178,4 +178,104 @@
     <item name="android:windowIsFloating">false</item>
     <item name="android:backgroundDimEnabled">true</item>
   </style>
+
+  <style name="H1Header" parent="Widget.AppCompat.TextView">
+    <item name="android:layout_width">wrap_content</item>
+    <item name="android:layout_height">wrap_content</item>
+    <item name="android:fontFamily">sans-serif</item>
+    <item name="android:textAllCaps">false</item>
```

Comment:
Most of these properties are the same, in that case, wouldn't it be better to make a common theme style and extend these themes from that?

Reply:
Done.